pkg/metrics/timer/timer_test.go

```diff
@@ -0,0 +1,130 @@
+package timer
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+func convertToFormat(buf bytes.Buffer) string {
+	output := buf.String()
+
+	outputText := ""
+
+	for _, item := range strings.Split(output, "\n") {
+		var dat map[string]interface{}
+
+		if err := json.Unmarshal([]byte(item), &dat); err != nil {
+			continue
+		}
+		outputText += dat["msg"].(string) + "\n"
+	}
+
+	return outputText
+}
+
+func TestBasicLogSummaryMultipleStages(t *testing.T) {
+	timer := NewTimer()
+
+	timer.StartTimer(TotalTimeElapsed)
+	timer.StartTimer("testStage1")
+	timer.StartTimer("testStage2")
+	timer.StartTimer("testStage3")
+	timer.StartTimer("testStage4")
+
+	// time.Sleep(5 * time.Second)
```

Comment:
I don't think we should keep commented code.

Reply:
Fixed it.